### ISSUE TITLE
Implement Admin Panel for Managing Events on Café Website (Frontend and Backend)

### ISSUE THREAD

**Issue by samar12-rad** [RamakrushnaBiswal/PlayCafe#91]
### Describe the feature

Currently, the café website lacks an admin interface for managing events. To streamline the process of adding and removing events, an admin panel should be created where the admin can easily manage events displayed on the website. This feature will allow the admin to create, update, and delete events dynamically.

Proposed Solution:

Create Admin Panel:
Develop a new admin panel section in the frontend where the admin can log in and manage events.

The admin should be able to:
Add new events (title, description, date, time, etc.)
Delete past events that are no longer relevant
Edit or update existing event details.

Backend API Integration:
Create backend routes and functionality to handle the addition, deletion, and updating of events.
Ensure proper authentication so that only authorized users (admins) can access these routes.

Frontend Design:

Design a user-friendly interface for the admin panel with clear input fields for event creation and options for managing events.
Incorporate status messages (e.g., success, error) for actions like adding, deleting, and editing events.

Admin Authentication:
Use Kinde auth to implement admin authentication and role-based access to ensure only admins can manage the events.

Benefits:
Allows the café owner to manage events without direct access to the codebase.
Makes it easier to keep the events section updated, showing only relevant and upcoming events.
Provides a dynamic solution that saves time and effort in managing event data.

### Add ScreenShots

![image](https://github.com/user-attachments/assets/ecd2d432-b4af-4ebb-b2f4-f2da70b5d9a5)

### Record

- [X] I agree to follow this project's Code of Conduct
- [X] I'm a gssoc-24-extd contributor
- [X] I want to work on this issue

**Comment by RamakrushnaBiswal** [comment]
@samar12-rad go for it. 
Can you fix this page UI ?

**Comment by samar12-rad** [comment]
@RamakrushnaBiswal , yes I can fix the ui for it? Can I please create an issue for that?

**Comment by RamakrushnaBiswal** [comment]
@samar12-rad sure

**Comment by samar12-rad** [comment]
@RamakrushnaBiswal assign me #98
